### PR TITLE
Roundstart Honkbot

### LINKED ISSUE #12604: Add a roundstart Honkbot that patrols the station
Honkbots are neat and not often seen.

### PR DESCRIPTION
**Pull request by carlopault**
image

# Changelog
:cl: Carlospaul
Added: roundstart honkbot
/:cl:

Associated feature request: https://github.com/HippieStation/HippieStation/issues/12604

## Why It's Good For The Game
"Honkbots are neat and not often seen."

### CODE REVIEW

**Comment by Neo-0** [comment]
so uhhh you're using the same branch #12646

**Comment by AnthonyJNgo** [comment]
You mixed this PR up with your chapel changes.

File: hippiestation/code/modules/clothing/shoes/shoes.dm [hunk 1]
```diff
@@ -19,3 +19,9 @@
 	icon = 'hippiestation/icons/mob/feet.dmi'
 	icon_state = "dress-white"
 	item_color = "dress-white"
+
+/obj/item/clothing/shoes/bronze/slow
```
Comment (AnthonyJNgo) [inline hippiestation/code/modules/clothing/shoes/shoes.dm:23]: This PR is just for adding a honkbot, but you can see here it is adding Ratvar's boots.

File: _maps/map_files/HippieStation/hippiestation.dmm [hunk 2]
```diff
@@ -60050,6 +60113,12 @@
 	},
 /turf/open/floor/plasteel,
 /area/hallway/primary/starboard)
+"vmn" = (
+/obj/machinery/door/morgue{
+	name = "Magical Item Storage"
```
Comment (AnthonyJNgo) [inline _maps/map_files/HippieStation/hippiestation.dmm:60118]: You committed the chapel commit to this PR also.

**Comment by AnthonyJNgo** [comment]
+3 for the honkbot, once you remove the other stuff.

**Comment by carlopault** [comment]
The last commit removed all the other changes, or at least should have. 

EDIT: Yes it did. This PR is clean and ready for merging.

**Comment by AnthonyJNgo** [comment]
It did not remove other changes. Go to "files changed" and you can see that your PR still has chapel code.

**Pull request closed without merging**